editor/editor_resource_picker.cpp

```diff
@@ -182,6 +181,17 @@ void EditorResourcePicker::_resource_saved(Object *p_resource) {
 	}
 }
 
+void EditorResourcePicker::_load_button_pressed() {
+	const Vector<String> &base_types_string = base_type.split(",");
+
+	Vector<StringName> base_types;
+	for (const String &type : base_types_string) {
+		base_types.push_back(type);
```

Comment:
```suggestion
	Vector<StringName> base_types;
	for (String type : base_type.split(",")) {
		base_types.push_back(type);
```
Would be cleaner IMO